IOU assert failed in PFS8

During the scenario run on 2020-05-03 PFS8 failed as the wrong IOU amount was detected.

```json
{
  "event": "Task errored",
  "logger": "scenario_player.tasks.base",
  "level": "error",
  "greenlet_name": "orchestration",
  "timestamp": "2020-05-04 01:02:22.444297",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 67, in __call__\n    return_val = self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/execution.py\", line 35, in _run\n    task()\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 67, in __call__\n    return_val = self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/execution.py\", line 35, in _run\n    task()\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 67, in __call__\n    return_val = self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 64, in _run\n    return self._process_response(response_dict)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/services.py\", line 356, in _process_response\n    f\"Expected amount of {exp_iou_amount} but got {actual_iou_amount}.\"\nscenario_player.exceptions.legacy.ScenarioAssertionError: Expected amount of 100 but got 200."
}
```

--- a/raiden/tests/scenarios/pfs8_mediator_goes_offline.yaml
+++ b/raiden/tests/scenarios/pfs8_mediator_goes_offline.yaml
@@ -116,7 +116,7 @@ scenario:
             - assert: {from: 2, to: 3, total_deposit: 1_000_000_000_000_000_000, balance: 999_000_000_000_000_000}
 
             # Check that IOU is created
-            - assert_pfs_iou: {source: 0, amount: {{ pfs_fee }}}
+            - assert_pfs_iou: {source: 0, amount: {{ 2 * pfs_fee }}}
       - serial:
           name: "Check that the [0, 1, 2, 3] path was used"
           tasks:

